### ISSUE TITLE
Support for GeoLocation interface

### ISSUE THREAD

**Issue by xs2bharat** [taimos/ask-sdk-test#11]
### Description

Any plan to add support for GeoLocation interface.

**Comment by hoegertn** [comment]
Definitely. Feel free to add any wishes to this ticket or any ideas about how the interface should look like. 

I will have a look at the GeoLocation interface, but as it is not yet ready in Germany, I did not have the chance to do so in the past.
